### PR TITLE
Feat/chatapp logout button

### PR DESCRIPTION
**Pull request by arnb-smnta**
This is the UI of the button it logs out and redirects to login page and cleares the local storages and cookies

### CODE REVIEW

**Comment by arnb-smnta** [comment]
closes #122

File: examples/apps/chat-app/web/react-vite-tailwind/src/pages/chat.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ import ChatItem from "../components/chat/ChatItem";
 import MessageItem from "../components/chat/MessageItem";
 import Typing from "../components/chat/Typing";
 import Input from "../components/Input";
-import { useAuth } from "../context/AuthContext";
+import { AuthContext, useAuth } from "../context/AuthContext";
```
Comment (jwala-anirudh) [inline examples/apps/chat-app/web/react-vite-tailwind/src/pages/chat.tsx:18]: why import `AuthContext` when there are no usages of it

**Comment by arnb-smnta** [comment]
@jwala-anirudh that was a mistake i overlooked I have deleted the import in the current commit

**Comment by arnb-smnta** [comment]
@jwala-anirudh should i add an alert on click logout for confirmation

**Pull request merged**